### ISSUE TITLE
Bug - Category filters do not work in Web App

### ISSUE THREAD

**Issue by cursemeslowly** [devRant/devRant#484]
As you can see at below screenshot, there are joke/meme posts showing on my feed although I have filtered them out.

![screenshot_2018-08-06 devrant - a fun community for developers to connect over code tech life as a programmer](https://user-images.githubusercontent.com/35447763/43700261-f8f90a62-9977-11e8-9793-77c4bb9b40be.png)

**Comment by Skayo** [comment]
Can reproduce. High priority

**Comment by TheOct0** [comment]
I have the same problem (forgive me for the duplicate, by the way)

I'll also add that rants I opened, upvoted and spent several minutes on still appear on my main feed. This seems to be a front feed issue, not just a tag issue. Feel free to tell me if those things are unrelated, but I felt like they were.

**Comment by DFoxinator** [comment]
Definitely need some more details on this - I'm not able to reproduce it. Do you have anything that could be blocking cookies? Or any specific reproduction steps?

**Comment by TheOct0** [comment]
Only third-party cookies are blocked on my browser @DFoxinator

**Comment by Demiko** [comment]
A bit off topic. I was going to create a bug because I could not find the filter preferences in web app. Those settings are in a bit obscure place.

**Comment by cursemeslowly** [comment]
Yes web app definitely has problem with category filter. I have tested it on Firefox private window. Still exist.

After logged in on private window, the category filter is "All" by default so I see few joke/meme rants. I have seen the same joke/meme rants when I use on my regular firefox with only "Rant and Questions" to show filter.

I updated the "All" filter to "Rant and Questions" on the private window. Few jokes/memes are disappeared, but not all. Still see some joke/meme rants though. I didn't subscribed to the ranters as well.

It seems to happen only on Algo mode. So maybe Algo is being prioritized over filter. Just a guess.

**Comment by 7twin** [comment]
@trogus @DFoxinator Was about to open a duplicate of this, still happens on firefox as @cursemeslowly described.

**Comment by DFoxinator** [comment]
Should be fixed now! Please let me know if you still see any issues. It seems there was a problem with the filters on logged-in only web algo feed view.

**Comment by 7twin** [comment]
@DFoxinator confirmed working!
